assignments/bmi_calculator/bmi_calculator.js

```diff
@@ -1,6 +1,16 @@
 // This function should return the BMI for a person 
 function BMICalculator(mass, height) {
     // Write your code here
+    if (mass<=0){
+        return "INVALID INPUT"
+    }
+    else if (height<=0){
+        return "INVALID INPUT"
+    }
+    else{
+        BMI = mass/(height*height)
+        return BMI
+    }
 }
```

Comment:
nicely executed, could've reduced the code lines by using || in the "if" statement itself